privaterelay/templates/includes/banners.html

```diff
@@ -79,28 +79,32 @@
             </div>
             {% endif %}
           {% elif not user_profile.subdomain %}
-            <div class="mpp-choose-subdomain" id="mpp-choose-subdomain">
-              <img src="/static/images/choose-subdomain.png" alt="">
-              <div class="border"></div>
-              <div class="mpp-choose-subdomain-form">
-                <h2>{% ftlmsg 'banner-choose-subdomain-headline-aliases' %}</h2>
-                <p>{% ftlmsg 'banner-choose-subdomain-copy' %}</p>
+            <div class="c-notification-banner subdomain-banner" id="mpp-choose-subdomain">
+              <div class="subdomain-banner-copy">
+                <div class="c-notification-banner-label">Action step</div>
+                <h2>Get a custom domain for your aliases</h2>
+                <div class="subdomain-banner-email-preview">***<span>@yourdomain</span>.relay.firefox.com</div>
```

Comment:
From the designs: 
<img width="484" alt="image" src="https://user-images.githubusercontent.com/2692333/138209614-a5f23579-be1d-4562-828d-dfe0bb899c0a.png">

This may be…a placeholder? Not really sure.

Reply:
Good flag.

Reply:
Given that the text below says `@yourdomain.mozmail.com`, surely that should be used here too?

Reply:
Also, though it probably won't have any practical consequences, because using new HTML elements is fun, this might be a good candidate for a [`<samp>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/samp)?

Reply:
@peiying2 When we convert this for localization – does it make sense to have "yourdomain" translated, but the rest to remain in English? We only allow ASCII (English) characters in the subdomain. (Outlier examples would be `Ñ`, etc.)